django/personnel/models.py

```diff
@@ -6,17 +6,20 @@
 
 
 class UserManager(DefaultUserManager):
-    # https://github.com/django/django/blob/master/django/contrib/auth/models.py#L131
+    # https://github.com/django/django/blob/master/django/contrib/auth/models.py#L128
```

Comment:
Not really aware of what this is doing, but i trust you.

Reply:
It's just a convenience link to the parent class source code since I reference it a lot